src/test/bridges/example/ExampleE2E.t.sol

```diff
@@ -47,20 +52,28 @@ contract ExampleE2ETest is BridgeTestBase {
 
         // Fetch the id of the example bridge
         id = ROLLUP_PROCESSOR.getSupportedBridgesLength();
+
+        // Subsidize the bridge when used with Dai
+        uint256 criteria = bridge.computeCriteria(USDC, USDC);
```

Comment:
The `computeCriteria` take addresses but `USDC` is an `IERC20`

Reply:
Addressed in cf8793d261feb6282b2a89f3dbc747281e16f21a